src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestActionMenu.tsx

```diff
@@ -53,6 +54,9 @@ const ChangeRequestActionMenu: React.FC<ChangeRequestActionMenuProps> = ({
     } else {
       try {
         await requestCRReview({ userIds: reviewerIds });
+        reviewerIds.forEach(function (value) {
+          toast.success('Request Review successful for reviewers ' + getUserFullName(reviewerIds[value]) + value);
```

Comment:
we're in the frontend, you can't call this method from the backend, they're two separate processes. You should instead use the list of all users (declared on line 46) and then for each reviewer Id, find the user which has the same id in that list and print their name